### PR TITLE
C6 get comments by article

### PR DESCRIPTION
**Pull request by elliejaneeeeee**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -150,6 +150,57 @@ describe('/api/articles', () => {
...
+        return request(app)
+        .get('/api/articles/notAnId/comments')
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe('400 Bad Request: Invalid Id')
+        })
+    })
+})
+
+
```
Comment (alice-y-codes) [inline __tests__/app.test.js:203]: Missing a test for status 200: Responds with empty array if article_id is valid but has no associated comments.

**Review by alice-y-codes — COMMENTED** [review]
Missing a test but the rest is perfecto
